UnitySDK/Assets/ML-Agents/Plugins/Barracuda.Core/Barracuda/Plugins/OSX/MacBLAS.cs

```diff
@@ -6,12 +6,12 @@
 
 
 [Preserve]
-public class MacBLAS : BLASPlugin
+public class MacBlas : BLASPlugin
```

Comment:
Not critical, but I'd leave this with the old capitalization (especially since it matches `BLASPlugin`)

Reply:
Hm, yeah the rest of this class had some specific capitalization for the function variable names.  I'll ping Mantas about that.

Reply:
Talked with Mantas. changes look good.  Renamed this back to what it was and updated the Rider config to Recognize `BLAS` as an abbreviation.